scala/core/src/sprites/Wire.scala

```diff
@@ -29,7 +27,8 @@ export WireProp._
   * where the beginning and the end are both horizontal, and it has no
   * arrowhead.
   */
-case class Wire() extends Sprite {
+case class Wire(src: Hom, tgt: Hom) extends Sprite {
```

Comment:
Why do we have `src` and `tgt` here? I try to separate out sprites from acsets, so that a sprite doesn't have to do any layout or query the acset; it gets everything it needs as properties.

Reply:
```quote
I try to separate out sprites from acsets, so that a sprite doesn't have to do any layout or query the acset; it gets everything it needs as properties.
```
Good to know. I'll keep this in mind.

I've refactored to add some additional props in the `wireProps`/`edgeProps` step and removed `src`/`tgt` from the sprites.